### DISCUSSION TITLE
Ενδιαμεση + τελικη παρουσιαση

### DISCUSSION THREAD

**Discussion by StavroulaAnatolaki** [courses-ionio/sw#1476] [Q&A] [answered]
Αν καποια παιδια δεν μπορουν να ερθουν για καμια παρουσιαση δια ζωσης τοτε αυτο θα επηρεασει τους βαθμους τους;

**Comment by p20sami** [comment]
Απ’οσο γνωρίζω πρέπει να γίνουν οι παρουσιάσεις για να γινει η αξιολόγηση της αναφοράς στο τέλος

**Comment by KonstantinosTourtsakis** [comment, marked as answer]
Οι δύο παρουσιάσεις (ενδιάμεση και τελική) είναι υποχρεωτικές και πρέπει να γίνουν διά ζώσης  #1143.

https://epidrome.github.io/teaching/progress/
> Η προφορική παρουσίαση (ενδιάμεσης και τελικής) προόδου είναι υποχρεωτική και πρέπει να γίνουν τουλάχιστον δύο εμπρόθεσμα ραντεβού.
